src/vm/runtimehandles.h

```diff
@@ -91,17 +91,17 @@ class LocalVariableInfo : Object
 
     REFLECTCLASSBASEREF GetType()
     {
-        return (REFLECTCLASSBASEREF)m_type;
+        return (REFLECTCLASSBASEREF)_type;
     }
 
     void SetType(OBJECTREF type)
     {
-        SetObjectReference(&m_type, type, GetAppDomain());
+        SetObjectReference(&_type, type, GetAppDomain());
     }
 
-    OBJECTREF m_type;
-    INT32 m_bIsPinned;
-    INT32 m_localIndex;
+    OBJECTREF _type;
+    BOOL _isPinned;
```

Comment:
This should be `CLR_BOOL`. Also, I would put it last - to avoid issues with padding.